### PR TITLE
Xenial - LANG env fix

### PR DESCRIPTION
**Pull request by zetneteork**
Adding env variable together with default variables.
```LC_ALL="en_US.UTF-8" \```
```echo -e 'LANG="en_US.UTF-8"\nLANGUAGE="en_US:en"\n' > /etc/default/locale && \```

### CODE REVIEW

**Comment by zetneteork** [comment]
This will fix issue with *Ombi*
[https://github.com/tidusjar/Ombi/issues/2917](https://github.com/tidusjar/Ombi/issues/2917)

**Comment by aptalca** [comment]
Looking into this, a couple of observations so far:
1) editing /etc/default/locale should not be necessary if locale is already set via variables (in the baseimage `locale` command already returns `en_US.UTF-8` for all vars except for LC_ALL

2) According to [this page](https://help.ubuntu.com/community/Locale#List_current_settings) `LC_ALL` is a hammer that overrides everything else, but without it, `LANG` value is used for any var that is not set

I'm not confident so far that these changes actually have an effect

File: Dockerfile [hunk 1]
```diff
@@ -87,7 +87,7 @@ RUN \
 	tzdata && \
  echo "**** generate locale ****" && \
  locale-gen en_US.UTF-8 && \
- echo -e 'LANG="en_US.UTF-8"\nLANGUAGE="en_US:en"\n' > /etc/default/locale && \
```
Comment (zetneteork) [inline Dockerfile]: **-e     enable interpretation of backslash escapes**
but test under `Dockerfile` show bad result. 
```
root@69b361d4ff3e:/# cat /etc/default/locale 
-e LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```

removing parameter `-e` fix it to 
```
LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```

**Review by zetneteork — COMMENTED** [review]
**-e     enable interpretation of backslash escapes**
but test under `Dockerfile` show bad result. 
```
root@69b361d4ff3e:/# cat /etc/default/locale 
-e LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```

removing parameter `-e` fix it to 
```
LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```

**Comment by aptalca** [comment]
Not needed, and doesn't solve the ombi issue: https://github.com/tidusjar/Ombi/issues/2917#issuecomment-481843703

closing

**Pull request closed without merging**